static/app/types/integrations.tsx

```diff
@@ -588,14 +588,16 @@ export type CodeownersFile = {
   raw: string;
 };
 
-export type FilesByRepository = {
-  [repoName: string]: {
-    [fileName: string]: {
-      authors?: {[email: string]: CommitAuthor};
```

Comment:
this is the only one i think is objectively worse

Reply:
because now its string? string? authors. a bit harder to figure out how to navigate the object

Reply:
@scttcper rev'd on the expression in use here.

```
type RepoName = string;
type FileName = string;
export type FilesByRepository = Record<
  RepoName,
  Record<
    FileName,
    {
    ...
```

could do the same kind of thing in other places too that used to be `{[foo: string]: ...}`